app/util/dappTransactions/index.test.ts

```diff
@@ -77,10 +83,44 @@ describe('Dapp Transactions utils :: validateTokenAmount', () => {
         TEST_GAS,
         TEST_FROM,
         TEST_SELECTED_ASSET,
+        TEST_SELECTED_ADDRESS,
         TEST_CONTRACT_BALANCES,
       ),
     ).toBeUndefined();
   });
+
+  it('should check value from contractBalances is selectedAddress is equal to from', async () => {
```

Comment:
```suggestion
  it('should check value from contractBalances if selectedAddress is from address', async () => {
```
I'm not sure how this checks the value from `contractBalances` since we check the value is equal to undefined. Should we mock a value and check that it is checked here?

Reply:
@digiwand : if validate returns undefined that implies that there is no error in value selected